Gemfile

```diff
@@ -25,8 +25,11 @@ gem 'coffee-rails', '~> 4.1.0'
 gem 'haml-rails'
 
 gem 'devise_token_auth'
+gem 'devise'
```

Comment:
de ce ai adaugat devise daca e deja definit in devise_token_auth ca dependenta? @ungureanu-bogdan-sorin iti cere activeadmin sa il adaugi?